src/client/collectors.py

```diff
@@ -62,34 +62,36 @@ def validate_condition(self, conditions: List[RuleCondition]):
 
 @inherit_docstrings
 class AcceptanceRate(CollectorConfig, spec_value=CollectorConfig.Type.ACCEPTANCE_RATE):
-    """Results of checking the answers of the Toloker.
+    """Counts accepted and rejected Toloker's assignments.
 
-    If non-automatic acceptance (assignment review) is set in the pool, add a rule to:
-    - Set the Toloker's skill based on their responses.
-    - Block access for Tolokers who give incorrect responses.
+    If non-automatic acceptance is set in the pool, you may use this collector to:
+    - Set a Toloker's skill.
+    - Block access for Tolokers with too many incorrect responses.
```

Comment:
```suggestion
    - Block access for Tolokers with too many rejected responses.
```
I guess "incorrect" is a bit misleading as the collector does not help you decide which responses are truly incorrect.